conda_devenv/devenv.py

```diff
@@ -12,12 +12,32 @@ def render_jinja(contents, filename, is_included):
     import sys
     import platform
 
+    iswin = sys.platform.startswith('win')
+    islinux = sys.platform.startswith('linux')
+    isosx = sys.platform.startswith('darwin')
+
+    is32bit = '32bit' == platform.architecture()[0]
+    is64bit = not is32bit
+
     jinja_dict = {
         "is_included": is_included,
         "os": os,
         "platform": platform,
         "root": os.path.dirname(os.path.abspath(filename)),
         "sys": sys,
+        "x86": 'x86' == platform.machine(),
+        "x86_64": 'x86_64' == platform.machine(),
+        "linux": islinux,
+        "linux32": islinux and is32bit,
+        "linux64": islinux and is64bit,
+        "armv6l": None,
```

Comment:
Do we want to let those with `None`? I see you didn't add them to the docs.
Could you remove them? And maybe create a issue do add them later?

Reply:
I'll see if I add them still in this PR. Before merging, I'll make a last call for approval just to be sure.

Reply:
I ended up deleting those three variables with None and entering a new Issue - #77 .